resolver-dns/src/main/java/io/netty/resolver/dns/DnsNameResolver.java

```diff
@@ -246,6 +248,7 @@ public DnsNameResolver(
         this.nameServerAddresses = checkNotNull(nameServerAddresses, "nameServerAddresses");
         this.queryTimeoutMillis = checkPositive(queryTimeoutMillis, "queryTimeoutMillis");
         this.resolvedAddressTypes = checkNonEmpty(resolvedAddressTypes, "resolvedAddressTypes");
+        this.resolvedAddressTypesEnum = resolvedAddressTypes(resolvedAddressTypes);
```

Comment:
I kept it when possible just so we not break user code if we can easily not do... so if possible yes please keep it if not possible break it